src/plugins/bubble/bubble.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * @title WET-BOEW Hello world plugin
+ * @overview Plugin contained to show an example of how to create your custom WET plugin
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @duboisp
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "wb-bubble",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+
+	/**
+	 * Initiate chat wizard bubble
+	 * @method initiateBubble
+	 * @param {jQuery DOM element} $selector Element which is the actual bubble
+	 */
+	initiateBubble = function( $selector ) {
+
+		var $footer = $( "#wb-info" );
+
+		// Hide basic form on load, show chat bubble instead
+		$selector.fadeIn( "slow" );
+
+		// Add some white space over the footer for the bubble to sit
+		$footer.addClass( componentName + "-mrgn" );
+
+		// Ensure that the bubble does not go passed the footer
+		if ( $footer.length ) {
+
+			// Keep the bubble sticky while scrolling Y until user reaches the footer
+			var stickyUntilFooter = function( $element ) {
+
+
+				// Equals to bubble default bottom value in CSS
+				var bottomY = 30;
+
+				if ( $( window ).scrollTop() >= $( document ).outerHeight() - $( window ).outerHeight() - $footer.outerHeight() ) {
+					$element.css( {
+						bottom: ( $footer.outerHeight() - ( $( document ).outerHeight() - $( window ).outerHeight() - $( window ).scrollTop() ) + bottomY )
+					} );
+
+					//console.log($element);
+				} else {
+					$element.css( {
+						bottom: bottomY
+					} );
+				}
+			};
+
+			// Correct bubble positionning on load, on resize an on Y scroll if necessary
+			stickyUntilFooter( $selector );
+
+			$( window ).on( "resize scroll", function() {
+				stickyUntilFooter( $selector );
+			} );
+		} else {
+
+			$( "main" ).after( "<footer id=\"wb-info\"></footer>" );
+			$( "#wb-info" ).addClass( componentName + "-mrgn" );
```

Comment:
Do line 67 and 68 in 1 call instead of 2. You can set CSS class when creating the footer element at line 67